### DISCUSSION TITLE
Enabling logging of current-db-size-in-use-bytes & current-db-size-in-use parameters during compaction run

### DISCUSSION THREAD

**Discussion by IamSatyaonline** [etcd-io/etcd#17042] [Q&A] [answered]
When ETCD runs defragmentation internally , ETCD print the values of parameters **current-db-size-in-use-bytes**, **current-db-size-in-use , current-db-size** etc. in ETCD logs to show the available size of DB and consumption of DB. We want to see the current DB usage of ETCD at run time whenever compaction runs internally by ETCD. Is it possible to print value of these parameters in ETCD logs during compaction ?

**Comment by IamSatyaonline** [comment]
Hi @ahrtr 
could you please help us here to enable the logging of current-db-size-in-use-bytes & current-db-size-in-use parameters during compaction ?
We see these log lines during defragmentation in etcd logs and same log lines we are expecting during compaction.
Thanks,
Satya

**Comment by jmhbnz** [comment, marked as answer]
Logging of these additional fields was enabled in https://github.com/etcd-io/etcd/pull/17204 and will be included in etcd `v3.6`.
